sky/task.py

```diff
@@ -207,7 +226,16 @@ def _validate(self):
                         f'a symlink to a directory). {self.workdir} not found.')
 
     @staticmethod
-    def from_yaml(yaml_path):
+    def from_yaml(yaml_path: str) -> 'Task':
+        """Initializes a task from a task YAML.
+
```

Comment:
nit: Can we add an example here:
```
Example

    task = sky.Task.from_yaml('/path/to/task.yaml')
```

Reason: one of our user created it with `sky.Task().from_yaml('task.yaml')`, which creates an unnecessary `Task` object

Reply:
Done, good call.